pytest_mypy/tests/test-extension.yml

```diff
@@ -3,5 +3,5 @@
         # if hook works, main should contain 'reveal_type(1)'
     reveal_type: 1
     out: |
-        main:1: note: Revealed type is 'builtins.int'
+        main:1: note: Revealed type is 'Literal[1]'
```

Comment:
Mentioned it in the commit, but I needed to change this so that the tests would pass on the latest mypy version (`0.750`). Was failing on Travis as well.